src/inverseRobot.js

```diff
@@ -7,7 +7,20 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const REVERSE_ROBOT = {};
```

Comment:
It's better to use camelCase construction, because in the form of uppercase letters, we explicitly set for some specific value that can be used globally